src/components/views/elements/Tooltip.tsx

```diff
@@ -155,11 +157,10 @@ export default class Tooltip extends React.Component<ITooltipProps> {
     }
 
     private renderTooltip = () => {
-        // Add the parent's position to the tooltips, so it's correctly
-        // positioned, also taking into account any window zoom
-        // NOTE: The additional 6 pixels for the left position, is to take account of the
-        // tooltips chevron
-        const style = this.updatePosition({});
+        let style: CSSProperties = {};
+        if(this.props.visible) {
```

Comment:
```suggestion
        // Only update the style if we need to for performance reasons
        if (this.props.visible) {
```

Reply:
@turt2live Added comments to both areas of optimizations 👍

You got to reviewing this before I finalized everything (the reviewer was stuck again). For reference, there is a before/after performance profile in the description now. Thanks for the review pass!